src/components/v2/V2Project/banners/RelaunchFundingCycleBanner/index.tsx

```diff
@@ -0,0 +1,233 @@
+    overflowAllowance: BigNumber.from(0), // nothing for the time being.
+    overflowAllowanceCurrency: BigNumber.from(0),
+  }
+
+  const launchFundingCycleTx = useLaunchFundingCyclesTx()
+
+  const loading =
+    payoutSplitsLoading ||
+    deprecatedFundingCycleLoading ||
+    tokenSplitsLoading ||
+    distributionLimitLoading
+
+  if (
+    !projectId ||
+    !deprecatedFundingCycle ||
+    !deprecatedFundingCycleMetadata
+  ) {
+    return null
+  }
+
+  const onLaunchModalOk = async () => {
+    const fundingCycleData: V2FundingCycleData = {
+      duration: newDuration ?? deprecatedFundingCycle.duration,
+      ballot: deprecatedFundingCycle.ballot,
+      weight: deprecatedFundingCycle.weight,
+      discountRate: deprecatedFundingCycle.discountRate,
+    }
+
+    const fundingCycleMetadata: V2FundingCycleMetadata =
+      deprecatedFundingCycleMetadata
+
+    await launchFundingCycleTx(
+      {
+        projectId,
+        fundingCycleData,
+        fundingCycleMetadata,
+        fundAccessConstraints: [deprecatedFundAccessConstraint],
+        groupedSplits: [
+          {
+            group: ETH_PAYOUT_SPLIT_GROUP,
+            splits: deprecatedPayoutSplits ?? [],
+          },
+          {
+            group: RESERVED_TOKEN_SPLIT_GROUP,
+            splits: deprecatedTokenSplits ?? [],
+          },
+        ],
+        mustStartAtOrAfter: newStart,
+      },
+      {
+        onDone() {
+          console.info('Transaction executed. Awaiting confirmation...')
+          setTransactionPending(true)
+        },
+        onConfirmed() {
+          setTransactionPending(false)
+          window.location.reload()
+        },
+      },
+    )
+  }
+
+  return (
+    <>
+      <Banner
+        title={<Trans>Funding cycle required.</Trans>}
+        body={
+          <Trans>
+            Your Juicebox project has no active funding cycle. Launch a funding
+            cycle to re-enable payments on your project.
+          </Trans>
+        }
+        actions={
+          <>
+            <Button type="primary" onClick={() => setModalOpen(true)}>
+              <Trans>Review and launch funding cycle</Trans>
+            </Button>
+          </>
+        }
+      />
+      <TransactionModal
+        visible={modalOpen}
+        title={<Trans>Launch funding cycle</Trans>}
+        okText={
+          <span>
+            <Trans>Launch funding cycle</Trans>
+          </span>
+        }
+        width={700}
+        onOk={onLaunchModalOk}
+        onCancel={() => setModalOpen(false)}
+        transactionPending={transactionPending}
+      >
+        {loading ? (
+          'Loading...'
+        ) : (
+          <>
+            <p style={{ marginBottom: '2rem' }}>
+              Relaunch your funding cycle on the new Juicebox V2 contracts.
+            </p>
+            <Form layout="vertical" style={{ marginBottom: '1rem' }}>
+              <div style={{ display: 'flex', gap: 20 }}>
+                <Form.Item
```

Comment:
yeah this isn't great, acknowledged.

For the Juicebox DAO case I wasn't sure if it will land on a specific day or if we need to input the exact timestamp.

Also with duration, unfortunately the time selector component isn't reusable, would need a refactor before using here. I think it's probably fine although not ideal - keeping in mind this is a once-off action for projects, and we can refine this modal if needed later.